Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -1121,10 +1192,13 @@ void FitPeaks::fitSpectrumPeaks(
       // point)
       cost =
           fitIndividualPeak(wi, peak_fitter, expected_peak_pos, peak_window_i,
-                            observe_peak_params, peakfunction, bkgdfunction);
+                            observe_peak_width, peakfunction, bkgdfunction);
       if (cost < 1e7) { // assume it worked and save out the result
```

Comment:
This should go after `processSinglePeakFitResult` which has additional checks for whether the fit was acceptable. Note that function does not change the value of cost passed to it, you need to do `fit_result->getCost(peak_index)`. This was something I didn't realise at the time of the original PR.